libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset.component.ts

```diff
@@ -69,6 +70,7 @@ export class SkyVerticalTabsetComponent
    * the tabset. This sets the tabset's `aria-labelledby` attribute
    * [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
    * If the tabset does not include a visible label, use `ariaLabel` instead.
+   * For more information about the `aria-labelledBy` attribute, see the [WAI-ARIA Definitions of States and Properties](https://www.w3.org/TR/wai-aria/#aria-labelledBy).
    */
   @Input()
```

Comment:
In the `ariaRole` property description for this component, we should also change the link to point to https://www.w3.org/WAI/PF/aria/#roles instead of https://www.w3.org/WAI/PF/aria/roles. ... Same thing for SkyVerticalTabComponent's `ariaRole` property.

Also, SkyVerticalTabComponent has an `ariaControls` property and a `tabId` property that both are ARIA-related. Do we need to make updates to those property descriptions?

Reply:
Updated the `ariaRole` and `ariaControls`. I am going to leave `tabId` as is as it was a little more complicated and it is deprecated anyway.